certbot/certbot/display/ops.py

```diff
@@ -241,12 +241,9 @@ def success_installation(domains):
 
     """
     z_util(interfaces.IDisplay).notification(
-        "Congratulations! You have successfully enabled {0}{1}{1}"
-        "You should test your configuration at:{1}{2}".format(
+        "Congratulations! You have successfully enabled {0}".format(
             _gen_https_names(domains),
-            os.linesep,
-            os.linesep.join(_gen_ssl_lab_urls(domains))),
-        pause=False)
+            pause=False))
```

Comment:
This parameter is now being passed to `format` rather than `notification`.

Reply:
Thanks!  Pushed a fix.